defaults.inc.bat

```diff
@@ -32,7 +32,7 @@ rem Comma separated list of build targets (default: Win64, Win32)
 if "%BUILD_TARGETS%" == ""                  set BUILD_TARGETS=Win64,Win32
 
 if "%PROJECT_PATH%" == ""                   set PROJECT_PATH=c:/Nextcloud/client-building
-if "%QT_PATH%" == ""                        set QT_PATH=c:/Qt/5.15.2
+if "%QT_PATH%" == ""                        set QT_PATH=d:/Qt/5.15.2
```

Comment:
@FlexW I guess we have come up to a conclusion that we must set QT_PATH from the script regardless if it was set before, to avoid having a necessity to change the environment variable on a build machine?

Reply:
might still be useful to force the path. I would just reset the environment variable on the build machine. The scripts in client-building use slashes everywhere, but I can change it.

Reply:
I am sure I have forgotten some stuff about `/` and `\` on Windows
still let's limit the changes to what is really needed to unblock us

Reply:
@matthieugallien what does that mean? Should I use or \ or / ?

Reply:
the use of `\` is confusing when looking at the whole file
I guess I disagree with @allexzander

Reply:
okay. I then revert it.

Reply:
@FlexW By resetting, do you mean removing QT_PATH from the build machine? I was not able to set it to an empty value, so, probably will have to remove it. Hope everything still works after that.

Reply:
@allexzander I mean removing it.

Reply:
Hi, the use of specific slashes / back slashes is quite important. Some tools involved in the build process don't like a specific type of slash, there are also cases where quotes / parameters or Windows-slash handling in environment variables cause a different behaviour. Also important for the scheduled build tasks. A lot of testing was done to make everything work and at least the slashes feel quite native to Git Bash. Windows is still very nasty with paths.

This might change in the future due to Windows updates (unlikely^^ maybe they improve it for WSL) or new requirements by the build tools involved, but I'd recommend to stick to the specific type of slash you found in the original `defaults.inc.bat`. Mixture is rare but was required to make the respective tool work.

Reply:
Thanks, @misch7 for the clarification.